tiket-service/Dockerfile

```
@@ -0,0 +1,20 @@
+FROM FROM maven:3.9.9-eclipse-temurin-21-alpine as BUILD-APPS
```

Hadolint (2.12.0)

[error] unexpected 'm'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)